databases/week3/mealSharingDB.sql

```diff
@@ -0,0 +1,45 @@
+CREATE DATABASE meal_sharing;
+
+USE meal_sharing;
+
+CREATE TABLE reservation (
+    reservationid INT(10) UNSIGNED NOT NULL auto_increment,
+    numberofguests INT (50) NOT NULL,
+    mealid INT (10) UNSIGNED,
+    createddate DATE,
+    contactphonenumber VARCHAR (20),
+    contactname VARCHAR (20),
+    contactemail VARCHAR (20),
```

Comment:
please use any of the standard naming conventions. It is really difficult to understand what the column names are :)
```suggestion
    reservation_id INT(10) UNSIGNED NOT NULL auto_increment,
    number_of_guests INT (50) NOT NULL,
    meal_id INT (10) UNSIGNED,
    created_date DATE,
    contact_phone_number VARCHAR (20),
    contact_name VARCHAR (20),
    contact_email VARCHAR (20),
```

Reply:
I am so sorry the this happened i used an online editor it somehow changed it. i implemented your feedback. Thank you so much for reviewing my hw.